docs/getting-started/publishing-keys.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+---
+
+# Key Publishing Guide
+
+This page explains how to configure the server and application to allow
+publishing keys.
```

Comment:
It might be helpful to have a short sentence or two before this. What is a "key" (there are many "keys" in this system)? What does one look like? What does it contain?

Reply:
I have expanded the titles of both publish and retrieval. Also going to disambiguate keys.